### PR TITLE
Adding logic to update event stats to aetos

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:
- Adding logic to update event stats to aetos
- Spec for kubevirt applications
- fix for PX namespace when using custom namespace.

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by vprabhakar-px — APPROVED** [review]
LGTM

File: tests/testTriggers.go [hunk 1]
```diff
@@ -9005,6 +9175,132 @@ func TriggerOCPStorageNodeRecycle(contexts *[]*scheduler.Context, recordChan *ch
...
+					time.Sleep(1 * time.Minute)
+
+					// Start NFS server to avoid pods stuck in terminating state (PWX-24274)
+					err = Inst().N.Systemctl(*n, "nfs-server.service", node.SystemctlOpts{
+						Action: "start",
+						ConnectionOpts: node.ConnectionOpts{
+							Timeout:         5 * time.Minute,
+							TimeBeforeRetry: 10 * time.Second,
+						}})
+					log.FailOnError(err, "Failed starting nfs service on node : %s", n.Name)
```
Comment (kphalgun-px) [inline tests/testTriggers.go:9278]: UpdateOutcome?

File: tests/testTriggers.go [hunk 2]
```diff
@@ -9005,6 +9175,132 @@ func TriggerOCPStorageNodeRecycle(contexts *[]*scheduler.Context, recordChan *ch
...
+					Step(fmt.Sprintf("wait until volume %v status is Up", vol.ID), func() {
+						log.InfoD("wait until volume %v status is Up", vol.ID)
+
+						t := func() (interface{}, bool, error) {
+							connOpts := node.ConnectionOpts{
+								Timeout:         1 * time.Minute,
+								TimeBeforeRetry: 5 * time.Second,
+								Sudo:            true,
+							}
+							cmd := fmt.Sprintf("pxctl volume inspect %s | grep \"Replication Status\"", vol.ID)
```
Comment (kphalgun-px) [inline tests/testTriggers.go:9228]: How about we use InspectVolume?

File: tests/testTriggers.go [hunk 3]
```diff
@@ -9005,6 +9175,132 @@ func TriggerOCPStorageNodeRecycle(contexts *[]*scheduler.Context, recordChan *ch
...
+						UpdateOutcome(event, err)
+					})
+
+					dashStats := make(map[string]string)
+					dashStats["node"] = n.Name
+					dashStats["volume"] = vol.Name
+					updateLongevityStats(ReallocateSharedMount, stats.PXRestartEventName, dashStats)
+
+					err = Inst().S.DisableSchedulingOnNode(*n)
+					dash.VerifySafely(err == nil, true, fmt.Sprintf("Disable sceduling on node : %s", n.Name))
```
Comment (kphalgun-px) [inline tests/testTriggers.go:9252]: UpdateOutcome?

File: tests/testTriggers.go [hunk 4]
```diff
@@ -9005,6 +9175,132 @@ func TriggerOCPStorageNodeRecycle(contexts *[]*scheduler.Context, recordChan *ch
...
+						StartVolDriverAndWait([]node.Node{*n})
+						err = Inst().S.EnableSchedulingOnNode(*n)
+						UpdateOutcome(event, err)
+
+						log.InfoD("validating applications")
+
+					}
+				}
+			}
+			ValidateApplications(*contexts)
```
Comment (kphalgun-px) [inline tests/testTriggers.go:9298]: How about we put the log statement and function call together?

File: tests/basic/vcluster_test.go [hunk 5]
```diff
@@ -1265,6 +1265,20 @@ var _ = Describe("{AutopilotMultipleFioOnManyVclusters}", func() {
 	})
 })
 
+var _ = Describe("{DeployMultipleKubevirtApps}", func() {
```
Comment (kphalgun-px) [inline tests/basic/vcluster_test.go:1268]: This is a reminder to remove this before merging the PR if it is not an actual test case.

File: drivers/scheduler/k8s/k8s.go [hunk 6]
```diff
@@ -926,7 +926,7 @@ func (k *K8s) Schedule(instanceID string, options scheduler.ScheduleOptions) ([]
 		if len(options.TopologyLabels) > 1 {
 			rotateTopologyArray(&options)
 		}
-
+                appNamespace = fmt.Sprintf("%v-%v", appNamespace, time.Now().Unix())
```
Comment (kphalgun-px) [inline drivers/scheduler/k8s/k8s.go:929]: How about we log the value of appNamespace?

File: tests/testTriggers.go (hunk 1 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go]: fixed

File: tests/testTriggers.go (hunk 3 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go:9252]: verifysafely is fine.. it wont end the test

File: tests/testTriggers.go (hunk 2 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go:9228]: volumeinspect sometimes takes time to reflect the actual status, hence used cli here

File: tests/testTriggers.go (hunk 4 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go]: fixed

File: tests/basic/vcluster_test.go (hunk 5 above)
Comment (lsrinivas-pure) [reply tests/basic/vcluster_test.go:1268]: time being we need it so keeping it

**Review by dbhatnagar-px — APPROVED** [review]
Lgtm

**Pull request merged**